python_modules/dagster/tox.ini

```diff
@@ -12,7 +12,8 @@ passenv = CI_* COVERALLS_REPO_TOKEN AWS_SECRET_ACCESS_KEY AWS_ACCESS_KEY_ID BUIL
 deps =
   scheduler_tests_old_pendulum: pendulum<2
   definitions_tests_old_pendulum: pendulum<2
-  storage_tests_old_sqlalchemy: sqlalchemy==1.3.24
+  storage_tests_sqlalchemy_1_3: sqlalchemy==1.3.24
+  storage_tests_sqlalchemy_1_4: sqlalchemy==1.4.48
```

Comment:
another option would be to make this sqlalchemy<1.4 and sqlalchemy<2.0 so we catch any patch updates